src/frontend/webpack.config.js

```diff
@@ -91,6 +91,13 @@ module.exports = (env) => {
             },
           ],
         },
+        {
+          test: /\.(jpg|png|svg)$/,
+          loader: 'file-loader',
+          options: {
+            name: '[path][name].[hash].[ext]',
+          },
+        },
```

Comment:
Should we use this loader only in dev mode ?

Reply:
it depend if we want to be able to add images asset directly i a component.
If not, i can look at moving it to the dev config.

Reply:
Right, let's configure things like that for now then we will rework if needed.